dbms/src/Columns/IColumn.h

```diff
@@ -24,6 +24,20 @@ namespace ErrorCodes
 class Arena;
 class ColumnGathererStream;
 
+struct ColumnSize
```

Comment:
It is specific for MergeTree. But we use only uncompressed size.
Maybe it's better to move this struct to Storage.h? (at least).